### PR TITLE
Add handling of em/rem/px units for cell_height and vertical_margin

### PR DESCRIPTION
**Pull request by jlowcs**
Add handling of em/rem/px units for cell_height and vertical_margin as well as no cell_height option/full CSS heights.

The point is mostly to be able to work with "em" sizes.

See changes in README for details.

### CODE REVIEW

File: src/gridstack.js [hunk 1]
```diff
@@ -945,15 +982,55 @@
...
+
+        var heightData = parseHeight(val);
+
+        if (this.opts.vertical_margin_unit === heightData.unit && this.opts.height === heightData.height) {
+            return ;
+        }
+        this.opts.vertical_margin_unit = heightData.unit;
+        this.opts.vertical_margin = heightData.height;
+
+        !noUpdate && this._update_styles();
```
Comment (kdietrich) [inline src/gridstack.js]: Please, no. We can definitely bear one more line to increase readability. ;)

Comment (kdietrich) [inline src/gridstack.js]: `!noUpdate && this._update_styles();`
Please, no. We can definitely bear one more line to increase readability. ;)

File: src/gridstack.js [hunk 2]
```diff
@@ -945,15 +982,55 @@
...
+
+        var heightData = parseHeight(val);
+
+        if (this.opts.cell_height_unit === heightData.height_unit && this.opts.height === heightData.height) {
+            return ;
+        }
+        this.opts.cell_height_unit = heightData.unit;
+        this.opts.cell_height = heightData.height;
+
+        !noUpdate && this._update_styles();
```
Comment (kdietrich) [inline src/gridstack.js]: `!noUpdate && this._update_styles();`
Same as above.

**Comment by jlowcs** [comment]
Better? :)

**Pull request merged**